src/main/java/org/codeforamerica/shiba/output/applicationinputsmappers/UtilityPaymentsInputsMapper.java

```diff
@@ -0,0 +1,93 @@
+package org.codeforamerica.shiba.output.applicationinputsmappers;
+
+import static org.codeforamerica.shiba.application.parsers.ApplicationDataParser.Field.UTILITY_PAYMENTS;
+import static org.codeforamerica.shiba.application.parsers.ApplicationDataParser.getValues;
+import static org.codeforamerica.shiba.output.ApplicationInputType.ENUMERATED_SINGLE_VALUE;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.codeforamerica.shiba.application.Application;
+import org.codeforamerica.shiba.output.ApplicationInput;
+import org.codeforamerica.shiba.output.Document;
+import org.codeforamerica.shiba.output.Recipient;
+import org.codeforamerica.shiba.pages.data.PagesData;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UtilityPaymentsInputsMapper implements ApplicationInputsMapper {
+
+  private static final List<String> EXPEDITED_UTILITY_PAYMENTS = List.of("HEATING", "COOLING",
+      "ELECTRICITY", "PHONE");
+  private static final List<String> HEATING_OR_COOLING = List.of("HEATING", "COOLING");
+  private static final List<String> WATER_OR_SEWER = List.of("WATER", "SEWER");
+
+  @Override
+  public List<ApplicationInput> map(Application application, Document _document,
+      Recipient _recipient, SubworkflowIterationScopeTracker _scopeTracker) {
+    return map(application.getApplicationData().getPagesData());
+  }
+
+  private List<ApplicationInput> map(PagesData pagesData) {
+    // Question was unanswered
+    if (pagesData.get("utilityPayments") == null) {
+      return Collections.emptyList();
+    }
+
+    List<ApplicationInput> results = new ArrayList<>();
+
+    List<String> utilityPayments = getValues(pagesData, UTILITY_PAYMENTS);
+
+    // No expedited utilities selected
+    if (utilityPayments.stream().noneMatch(EXPEDITED_UTILITY_PAYMENTS::contains)) {
+      results.add(new ApplicationInput("utilityPayments", "noExpeditedUtilitiesSelected",
+          "true", ENUMERATED_SINGLE_VALUE));
+      results.add(new ApplicationInput("utilityPayments", "NO_EXPEDITED_UTILITIES_SELECTED",
+          "NO_EXPEDITED_UTILITIES_SELECTED", ENUMERATED_SINGLE_VALUE));
+    }
```

Comment:
Why are we having to add two very similar things here? Not saying it's bad necessarily but I'm a bit confused.

Is this related to that `OneToManyApplicationInputsMapper`? It might be nice to have a few more comments explaining what's happening in this class generally

Reply:
Yeah, this is pdf vs xml (unrelated to OneToMany mapper). So PDF generator interprets `ENUMERATED_MULTI_VALUE` as "Yes" if the field is there, so it will do the "COOKING_FUEL" -> "Yes" on the acroform. XML doesnt do that but it replaces the values with the enums it has listed. So "COOKING_FUEL" -> "Cooking Fuel". Having different values makes sense because the XML doesnt have the radio buttons to write "Yes". https://github.com/codeforamerica/shiba/blob/main/src/main/java/org/codeforamerica/shiba/output/pdf/BinaryPdfField.java
I'm going to add this to the "pain points" doc.

Reply:
Can you add that explanation as a comment in the mapper? that's great and makes a lot more sense